src/function/scalar/string/suffix.cpp

```diff
@@ -0,0 +1,42 @@
+#include "duckdb/function/scalar/string_functions.hpp"
+#include "duckdb/common/types/string_type.hpp"
+
+#include "duckdb/common/exception.hpp"
+
+using namespace std;
+
+namespace duckdb {
+
+static bool suffix(const string_t &str, const string_t &suffix);
+
+struct SuffixOperator {
+	template <class TA, class TB, class TR> static inline TR Operation(TA left, TB right) {
+		return suffix(left, right);
+	}
+};
+
+static bool suffix(const string_t &str, const string_t &suffix) {
+    auto suffix_size = suffix.GetSize();
+    auto str_size = str.GetSize();
+    if(suffix_size > str_size)
+        return false;
```

Comment:
Could you put brackets here? We prefer not to have the single-line after if statements because they have the potential of causing bugs later. i.e. change to the following please:

```cpp
if (suffix_size > str_size) {
         return false;
}
```